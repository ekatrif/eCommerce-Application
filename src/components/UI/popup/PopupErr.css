.popupErr {
  position: fixed;
  left: calc(50% - 100px);
  bottom: 125px;
  display: flex;
  justify-content: center;
  align-items: center;
  width: 200px;
  height: 50px;
  color: #fff;
  background-color: #953027;
  box-shadow: 0px 0px 20px #1F2827;
  transition: all .3s ease;
  scale: 0;
}

.popupErr-active {
  scale: 1;
}