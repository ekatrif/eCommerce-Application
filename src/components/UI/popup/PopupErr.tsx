import React from 'react';
import './PopupErr.css';

interface IPopupProps {
  active: boolean;
  setActive: React.Dispatch<React.SetStateAction<boolean>>;
}

const PopupErr: React.FC<IPopupProps> = ({ active, setActive }) => {
  return (
    <div className={active ? 'popupErr-active popupErr' : 'popupErr'} onClick={(): void => setActive(false)}>
      <p>The user already exists!</p>
    </div>
  );
};

export default PopupErr;
