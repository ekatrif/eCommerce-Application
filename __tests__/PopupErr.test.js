import React from 'react';
import renderer from 'react-test-renderer';
import PopupErr from '../src/components/UI/Popup/PopupErr';

test('PriceSlider renders correctly', () => {
  const tree = renderer
    .create(<PopupErr />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});