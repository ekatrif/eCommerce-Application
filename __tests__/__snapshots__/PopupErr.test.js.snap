// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PriceSlider renders correctly 1`] = `
<div
  className="popupErr"
  onClick={[Function]}
>
  <p>
    The user already exists!
  </p>
</div>
`;
